src/guide/jbs-jdk-bug-system.md

```diff
@@ -316,10 +295,18 @@ This table contains some frequently used JBS labels and their meaning. Please he
       A test that should be platform agnostic but is consistently failing on a specific OS would for instance be labeled with [intermittent-environment]{.jbs-label}, while a test that fails every time it's run on some specific hardware would be labeled with [intermittent-hardware]{.jbs-label}.
     </td>
   </tr>
+  <!-- J -->
   <tr>
+    <td class="dictionary">[[jvmci]{.jbs-label}]{#jvmci}</td>
     <td class="dictionary">
-      [[maintainer-pain]{.jbs-label}]{#maintainer-pain}
+      Used to identify issues in the JVM Compiler Interface.
     </td>
+  </tr>
+  <!-- K -->
+  <!-- L -->
+  <!-- M -->
```

Comment:
What might this be?

Reply:
Markers for the future addition of labels beginning with `K`, `L`, and `M`. I think it should remain as it is in keeping with similar additions such as those on lines 298 and 328.  I imagine that the `l10n`-prefixed labels may be candidates at some point later.

Reply:
What Iris said :-)  Navigation markers to make it easier to find the right place when scrolling through the file. I found them very helpful when I prepared this patch.